### PR TITLE
1360 facilities grid

### PR DESCRIPTION
**Pull request by Sepehr-Sobhani**
[ISSUE 1360](https://github.com/bcgov/cas-registration/issues/1360)

TEST:
- Log in as CAS Admin or `bc-cas-dev` and go to the following link to view all facilities related to the operation with UUID `002d5a9e-32a6-4191-938c-2c02bfec592d`.
- http://localhost:3000/registration/operations/002d5a9e-32a6-4191-938c-2c02bfec592d/facilities

### CODE REVIEW

**Review by marcellmueller — COMMENTED** [review]
Nice work! It's been a long day and I need to take another look at this in the morning on a fresh head but just a few suggestions :)

File: bciers/apps/registration/tests/facilities/Facilities.test.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import { render, screen } from "@testing-library/react";
+import {
+  fetchFacilitiesPageData,
```
[the remark this answers is not in the document]
Comment (Sepehr-Sobhani) [reply bciers/apps/registration/tests/facilities/Facilities.test.tsx:3]: I wasn't sure too :) I tried to follow the existing pattern, but if that doesn't make sense, I can mock it in the place it's being used.

**Review by marcellmueller — COMMENTED** [review]
@Sepehr-Sobhani thanks for the thorough review! I implemented most of your suggestions :)

**Review by marcellmueller — APPROVED** [review]
Nice work!! Really sorry I missed your comment saying this was ready yesterday. I hope rebasing isn't too difficult 😿 

I think the main thing will be updating the files in the route since I removed `registration/dashboard` route.

Also, you could update the Note component now if you wish!

**Pull request merged**